### PR TITLE
ci: add azure pipeline

### PR DESCRIPTION
**Pull request by KazuCocoa**
Just adds `npm run test` for now

### CODE REVIEW

File: azure-templates/node_setup_steps.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+steps:
```
Comment (mykola-mokhnach) [inline azure-templates/node_setup_steps.yml:1]: we could make `azure-templates` folder hidden (e.g. rename it to `.azure-templates`)

File: .azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+stages:
+- stage: Unit_Tests_And_Linters
+  jobs:
+  - job: Node_Unit_Tests
+    steps:
+    - template: azure-templates/node_setup_steps.yml
+    - script: npm install
+    - script:
+        npm run test
```
Comment (mykola-mokhnach) [inline .azure-pipelines.yml:18]: this could be in the same row with `script:`

Comment (mykola-mokhnach) [inline .azure-pipelines.yml:18]: should we also run `lint` task or it is already included?

Comment (KazuCocoa) [reply .azure-pipelines.yml:18]: The output had `lint`, so this should have

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**